test/test_parameters.py

```diff
@@ -0,0 +1,29 @@
+from tonic.parameters import Parameter, ChildParameter
+
+
+def test_addition_1():
+    p1 = Parameter(0.0, is_variable=False)
+    p2 = Parameter(1.0)
+    assert p1 + p2 == 1.0
+    assert p1 - p2 == -1.0
+
```

Comment:
What happens when we add parameters with different values of `is_variable`? It would be good to check  `(p1 + p2).is_variable` is set correctly.

Reply:
Would be good to check that p1 and p2 get assigned the correct names in the namespace here as well